interfaces/cython/cantera/ctml_writer.py

```diff
@@ -2295,15 +2304,14 @@ def __init__(self,
                         initial_state, options)
         self._tr = transport
         self._n = site_density
-        self._species = species
+
```

Comment:
As minor as it is, I'd suggest that this fix for `lattice` be done in a separate commit, since it serves a distinct purpose, and 'Enable instantiation of lattice class in CTI files` is a nice thing to see in the commit log.